bootcamp/tutorials/quickstart/hybrid_search_with_milvus.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Insert documents and their embeddings into the collection
</pre><p>Insert documents and their embeddings into the collection. For efficiency, we insert 50 records in each small batch.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>